src/lib/getSchemaUi.ts

```
@@ -0,0 +1,228 @@
+
+      properties.push({
+        name: 'additionalProperties',
+        types: [additionalProps.type as JSONSchemaType],
+        required: false,
+        meta: { isAdditionalProperties: true },
+      })
+    }
+
+    return properties
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)